armedilla_alliyah_samantha/index.html

```diff
@@ -25,7 +25,7 @@ <h2>My interest and hobbies</h2>
         <div class="segment">
           <h2>My Works</h2>
```

Comment:
Also you can add 1 extra new line for readability purpose. Not just in that line but also in other parts of your index.html.

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d18931cb7_3_0